lib/ansible/plugins/connection/winrm.py

```diff
@@ -215,6 +214,7 @@ def _connect(self):
 
     def exec_command(self, cmd, in_data=None, sudoable=True):
         super(Connection, self).exec_command(cmd, in_data=in_data, sudoable=sudoable)
+        cmd = self._shell.join_cmd_list(cmd)
         cmd_parts = shlex.split(to_bytes(cmd), posix=False)
         cmd_parts = map(to_unicode, cmd_parts)
```

Comment:
Just at a glance, shouldn't we be able to to_unicode the list directly instead of joining + split?